docker-registry-sync/reposync/setup.py

```diff
@@ -1,10 +1,11 @@
+import os
 import re
 import sys
 from pathlib import Path
 
 from setuptools import find_packages, setup
 
-current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
```

Comment:
IMO the previous way of getting ``current_dir`` is far more general and has no overhead

Reply:
I had to replace while wrestling with tox and forgot to bring it back :\